packages/rum-core/src/domain/rumEventsCollection/longTask/longTaskCollection.ts

```diff
@@ -19,7 +19,7 @@ export function startLongTaskCollection(lifeCycle: LifeCycle) {
     lifeCycle.notify(LifeCycleEventType.RAW_RUM_EVENT_COLLECTED, {
       rawRumEvent,
       startTime: startClocks.relative,
-      domainContext: { performanceEntry: (entry as unknown) as PerformanceEntry },
+      domainContext: { performanceEntry: entry instanceof PerformanceEntry ? entry.toJSON() : entry },
```

Comment:
Could we receive different representation for long tasks?

Reply:
Yes, during tests we mock performence entries so they don't have a `toJSON` method. Do you think we should implement a `toJSON` method for `RumPerformanceLongTaskTiming`?

Reply:
oh ok, then I'd be in favor of having the test code closer to the prod behavior rather than supporting test behavior in prod code

Reply:
Ok I'll add the `toJSON` method to `RumPerformanceLongTaskTiming` then